net-voip/captagent/captagent-6.3.1.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
+
+DESCRIPTION="CaptAgent:  100% Open-Source Packet Capture Agent for HEP"
+HOMEPAGE="https://sipcapture.org/"
+SRC_URI="https://github.com/sipcapture/captagent/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="ipv6 mysql pcre redis ssl uv"
```

Comment:
I can't see `uv` having a function in this ebuild.

Reply:
In DEPEND ... but that should be associated with a ./configure option which there is not ... rechecking ./configure, it's a hard dep, moving that accordingly and dropping the IUSE.